tests/v2/validation/charts/cisbenchmark_test.go

```diff
@@ -0,0 +1,339 @@
+	c.session.Cleanup()
+}
+
+func (c *CisBenchmarkTestSuite) SetupSuite() {
+	testSession := session.NewSession()
+	c.session = testSession
+
+	client, err := rancher.NewClient("", testSession)
+	require.NoError(c.T(), err)
+
+	c.client = client
+
+	cisConfig := new(CisConfig)
+	config.LoadConfig(CisConfigFileKey, cisConfig)
+	c.cisProfileName = cisConfig.ProfileName
+	require.NotEmptyf(c.T(), c.cisProfileName, "CIS profile name required")
+
+	clusterName := client.RancherConfig.ClusterName
+	require.NotEmptyf(c.T(), clusterName, "Cluster name to install is not set")
+
+	clusterID, err := clusters.GetClusterIDByName(client, clusterName)
+	require.NoError(c.T(), err)
+
+	if cisConfig.ChartVersion == "" {
+		cisConfig.ChartVersion, err = client.Catalog.GetLatestChartVersion(charts.CISBenchmarkName, catalog.RancherChartRepo)
+		require.NoError(c.T(), err)
+	}
+
+	projectConfig := &management.Project{
+		ClusterID: clusterID,
+		Name:      CISBenchmarkProjectName,
+	}
+	createdProject, err := client.Management.Project.Create(projectConfig)
+	require.NoError(c.T(), err)
+	require.Equal(c.T(), createdProject.Name, CISBenchmarkProjectName)
+	c.project = createdProject
+
+	c.chartInstallOptions = &charts.InstallOptions{
+		Cluster: &clusters.ClusterMeta{
+			ID:   clusterID,
+			Name: clusterName,
+		},
+		Version:   cisConfig.ChartVersion,
+		ProjectID: createdProject.ID,
+	}
+
+}
+
+func (c *CisBenchmarkTestSuite) TestFreshInstallCisBenchmarkChart() {
+	subSession := c.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := c.client.WithSession(subSession)
+	require.NoError(c.T(), err)
+
+	c.T().Log("Installing latest version of cis benchmark chart")
+	err = charts.InstallCISBenchmarkChart(client, c.chartInstallOptions)
+	require.NoError(c.T(), err)
+
+	c.T().Log("Waiting for cisbenchmark chart deployments to have expected number of available replicas")
+	err = extencharts.WatchAndWaitDeployments(client, c.project.ClusterID, charts.CISBenchmarkNamespace, metav1.ListOptions{})
+	require.NoError(c.T(), err)
+
+	scanName := namegen.AppendRandomString("scan")
+
+	c.T().Log("Running a scan")
+
+	// Run the scan
+	err = runScan(client, scanName, c.cisProfileName, c.project.ClusterID)
+	require.NoError(c.T(), err)
+
+}
+
+func (c *CisBenchmarkTestSuite) TestUpgradeCisbenchmarkChart() {
+	subSession := c.session.NewSession()
+	defer subSession.Cleanup()
+
+	client, err := c.client.WithSession(subSession)
+	require.NoError(c.T(), err)
+
+	// Change cisbenchmark install option version to previous version of the latest version
+	versionsList, err := client.Catalog.GetListChartVersions(charts.CISBenchmarkName, catalog.RancherChartRepo)
+	require.NoError(c.T(), err)
+
+	if len(versionsList) < 2 {
+		c.T().Skip("Skipping the upgrade case, only one version of cis-benchmark is available")
+	}
+	assert.GreaterOrEqualf(c.T(), len(versionsList), 2, "There should be at least 2 versions of the cis-benchmark chart")
```

Comment:
is this assertion needed? provided the test will be skipped if there are less than 2 versions.